docs/product/performance/assets.mdx

```diff
@@ -1,20 +1,24 @@
 ---
-title: Resources
+title: Assets
 sidebar_order: 70
-description: "Learn more about browser resource performance monitoring, which allows you to debug the performance of loading JavaScript and CSS on your frontend."
+description: "Learn more about browser asset performance monitoring, which allows you to debug the performance of loading JavaScript and CSS on your frontend."
 ---
 
-If you have performance monitoring enabled for your frontend, you can see how your browser resources are performing in Sentry.
+<Note>
+This product feature was originally named "Resources", "Assets" has the same functionality with a new exciting name.
+</Note>
+
+If you have performance monitoring enabled for your frontend, you can see how your browser assets are performing in Sentry.
 
-Starting with the [**Resources** page](#resources-page), you get a high-level overview of how your resources are doing. From there, you can drill into a specific resource's [**Resource Summary** page](#resource-summary-page) and then investigate sample events from the [Sample List](#sample-list) to better understand the context of its performance in a specific page.
+Starting with the [**Assets** page](#assets-page), you get a high-level overview of how your assets are doing. From there, you can drill into a specific asset's [**Asset Summary** page](#asset-summary-page) and then investigate sample events from the [Sample List](#sample-list) to better understand the context of its performance in a specific page.
 
 <div style={{position: 'relative', paddingBottom: 'calc(50.86013462976814% + 41px)', height: 0}}><iframe src="https://demo.arcade.software/BxySgGk7hpJ72xquLdKe?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: 'absolute', top: 0, left: 0, width: '100%', height: '100%', colorScheme: 'light'}}></iframe></div>
```

Comment:
```suggestion
<div style={{position: 'relative', paddingBottom: 'calc(50.86013462976814% + 41px)', height: 0}}><iframe src="https://demo.arcade.software/c7lgN0cFkg0R2tb9t0jH?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: 'absolute', top: 0, left: 0, width: '100%', height: '100%', colorScheme: 'light'}}></iframe></div>
```
Updated the arcade link! Will remove the other arcades as this one covers the entire workflow (going to do the same for the other modules as well in a separate PR)

Reply:
Perfect!